libr/include/r_core.h

```diff
@@ -330,6 +330,7 @@ typedef struct r_core_t {
 	bool log_events; // core.c:cb_event_handler : log actions from events if cfg.log.events is set
 	RList *ropchain;
 	bool use_tree_sitter_r2cmd;
+	RCharset custo;
```

Comment:
@trufae because there may be several RCharset for each native charset in the future example: ascii, utf8, etc... Then we add a new charset know as custom because it is made by the user or downloaded from internet.

Reply:
then name it custom?

Reply:
yes .rename it to custom please